build.gradle

```diff
@@ -102,16 +95,37 @@ docker {
     copySpec.from(tarTree("$buildDir/distributions/${applicationName}.tar"))
 }
 
-compileKotlin {
-    kotlinOptions {
-        jvmTarget = "11"
-        freeCompilerArgs += "-XXLanguage:+UnitConversion"
+tasks.withType(KotlinCompile).configureEach {
+    kotlinOptions.jvmTarget = "11"
```

Comment:
This block is deprecated in the new kotlin version. The `compilerOptions` should be used instead
```groovy
tasks.withType(KotlinCompile).configureEach {
    compilerOptions {
        jvmTarget.set(JvmTarget.JVM_11)
    }
}
```